pkg/scalers/externalscaler/externalscaler.proto

```diff
@@ -26,7 +26,10 @@ message GetMetricSpecResponse {
 
 message MetricSpec {
     string metricName = 1;
+
+    // deprecated, use targetSizeFloat instead
     int64 targetSize = 2;
+    double targetSizeFloat = 3;
```

Comment:
This actually breaks the GRPC contract as I explained here -> https://github.com/kedacore/keda/pull/6482#pullrequestreview-2559038091

Currently will work just because the server has been updated too, but for other proto files this breaks. Make all the new fields `optional` please.